addons/mail/models/discuss/discuss_channel.py

```diff
@@ -1193,7 +1193,7 @@ def _send_transient_message(self, partner_to, content):
     def execute_command_help(self, **kwargs):
         partner = self.env.user.partner_id
         if self.channel_type == 'channel':
-            msg = _("You are in channel <b>#%s</b>.", html_escape(self.name))
+            msg = _("You are in channel <b>#%s</b>.", "".join(html_escape(self.name)))
```

Comment:
join doesn't make sense when given a string as param?

Reply:
the `html_escape(self.name)` is returning a `Markup` not a string